server.py

```diff
@@ -5,11 +5,575 @@
 @author: Saniya
 """
 
-import http.server
-import socketserver
-
-PORT = 8080
-Handler = http.server.SimpleHTTPRequestHandler
-with socketserver.TCPServer(("", PORT), Handler) as httpd:
-    print("Serving at port: ", PORT)
-    httpd.serve_forever()
\ No newline at end of file
+import flask
+from flask import request, jsonify
+
+app = flask.Flask(__name__)
+app.config["DEBUG"] = True
+
+Port = 8000
+false = False
+true = True
+restaurants = [
```

Comment:
Extreme nit: If i wrote this, i would move these data sources to a separate file, so that this file is kept succinct. But since these are eventually going away, its up to you if you think it is worth changing.

Reply:
If I had them in a separate file: data.py, could I just do 

import restaurants from data.py

I would check but I'm not on my computer so I can't run it

Reply:
I think the python would be
from data import restaurants, reviews